app/javascript/components/shared_components/Logo.jsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import Image from 'react-bootstrap/Image';
+import PropTypes from 'prop-types';
 import useSiteSetting from '../../hooks/queries/site_settings/useSiteSetting';
 
-export default function Logo() {
+export default function Logo({ width }) {
```

Comment:
I'm not a big fan of this - passing the width in px isn't very friendly to developers. I would just add a size prop (s, m, l, xl) to keep consistency with how everything else is done

Reply:
Okay, I will see what I can do.